src/app/helptext/system/cloudcredentials.ts

```diff
@@ -266,6 +271,82 @@ a new application key, log in to the Backblaze account, go to the \
     )
   },
 
+  user_openstack_swift: {
+    placeholder: T('User Name'),
+    tooltip: T(''),
+  },
+
+  key_openstack_swift: {
+    placeholder: T('API Key or Password'),
+    tooltip: T('')
+  },
+
+  auth_openstack_swift: {
+    placeholder: T('Authentication URL'),
+    tooltip: T('')
+  },
+
+  user_id_openstack_swift: {
+    placeholder: T('User ID'),
+    tooltip: T('')
+  },
+
+  domain_openstack_swift: {
+    placeholder: T('User Domain'),
+    tooltip: T('')
+  },
+
+  tenant_openstack_swift: {
+    placeholder: T('Tenant Name'),
+    tooltip: T('')
+  },
+
+  tenant_id_openstack_swift: {
+    placeholder: T('Tenant ID'),
+    tooltip: T('')
+  },
```

Comment:
```
  tenant_id_openstack_swift: {
    placeholder: T('Tenant ID'),
    tooltip: T('Tenant ID - optional for v1 auth, this or tenant \
 required otherwise \
 <a href="https://rclone.org/swift/#standard-options" target="_blank">(rclone documentation)</a>.')
  },
```